commodore/helpers.py

```diff
@@ -175,14 +175,15 @@ def rm_tree_contents(basedir):
             os.unlink(f)
 
 
-def relsymlink(srcdir, srcname, destdir, destname=None):
-    if destname is None:
-        destname = srcname
+# pylint: disable=unsubscriptable-object
+def relsymlink(src: P, dest_dir: P, dest_name: Union[P, str] = None):
```

Comment:
I don't think having `dest_name` as a Path makes sense. I'd suggest making `dest_name` `Optional[str]` to allow `None` or strings.

```suggestion
def relsymlink(src: P, dest_dir: P, dest_name: Optional[str] = None):
```

If the type checker complains about assigning `src.name` to `dest_name` with the adjusted type hint, the assignment can be written as

```
dest_name = str(src.name)
```